src/api/config/environments/production.rb

```diff
@@ -71,6 +71,18 @@
     config.logger = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  # Use lograge to show the logs in one line
```

Comment:
maybe: _Use lograge to show **custom** logs in one line_

Reply:
and also in the commit message, because it is not only about rendering it in one line , but that you can decide what to render :wink:

Reply:
If I turn the log into a one liner... they are custom by definition... :smile:

Reply:
I don't thing that saying that the new way of logging is "custom" adds anything important really...